utils/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "validate-quickstart-install-plans": "ts-node validate-quickstart-install-plans.ts",
+    "validate-quickstart-data-sources": "ts-node validate-quickstart-data-sources.ts",
```

Comment:
Looks like the file is actually called `validate-quickstart-data-source.ts` (with out `s`), let's update the filename to be plural.